main.go

```diff
@@ -3,43 +3,38 @@ package main
 import (
 	"./metrics"
 	"./service"
-	"time"
+	"golang.org/x/net/context"
 )
 
 func main() {
-	logPrintf("Starting Docker Flow: Swarm Listener")
+	logPrintf("Starting Docker Flow Swarm Listener (now with Docker events)")
+
+	ctx := context.Background()
 	s := service.NewServiceFromEnv()
 	n := service.NewNotificationFromEnv()
 	serve := NewServe(s, n)
+
 	go serve.Run()
 
-	args := getArgs()
+	eventStream, err := s.GetEventStream()
```

Comment:
It might make sense to move everything from here until the end to a separate function in, for example, the `service` package. That will make it easier to test.